frontend/apps/reader/modules/readergesture.lua

```diff
@@ -1415,6 +1419,22 @@ function ReaderGesture:gestureAction(action, ges)
         self.ui.menu:exitOrRestart()
     elseif action == "restart" then
         self.ui.menu:exitOrRestart(function() UIManager:restartKOReader() end)
+    elseif action == "poweroff" then
+        UIManager:show(ConfirmBox:new{
+            text = _("Are you sure you want to power off the device?"),
+            ok_text = _("Power off"),
+            ok_callback = function()
+                UIManager:nextTick(UIManager.poweroff_action)
+            end,
+        })
+    elseif action == "reboot" then
```

Comment:
Bit of a nitpick that I didn't think of before @poire-z mentioned it, but the reboot/poweroff order also happens here